### PR TITLE
Project Setup

### PR DESCRIPTION
**Pull request by mcolombo87**
## Summary

Project Setup: Part of "Maquetado" training

## Trello Card

https://trello.com/c/0LiCH5AN/11-setup-del-proyecto

### CODE REVIEW

File: maquetado/scss/application.scss [hunk 1]
```diff
@@ -0,0 +1,4 @@
+.title {
+    color: red;
```
Comment (erikankodelia) [inline maquetado/scss/application.scss]: Only two spaces

File: maquetado/scss/application.scss [hunk 2]
```diff
@@ -0,0 +1,4 @@
...
+  }
```
Comment (erikankodelia) [inline maquetado/scss/application.scss]: Remove extra spaces

File: maquetado/scss/application.scss (hunk 1 above)
Comment (mcolombo87) [reply maquetado/scss/application.scss]: Done!

File: maquetado/scss/application.scss (hunk 2 above)
Comment (mcolombo87) [reply maquetado/scss/application.scss]: Done!

**Pull request merged**